pkg/protocol/http1/client.go

```diff
@@ -167,6 +169,20 @@ func (c *HostClient) WantConnectionCount() (count int) {
 	return c.connsWait.len()
 }
 
+func (c *HostClient) ConnPoolState() config.ConnPoolState {
+	c.connsLock.Lock()
+	defer c.connsLock.Unlock()
+	cps := config.ConnPoolState{
+		ConnNum: len(c.conns),
+		Host:    c.Addr,
```

Comment:
Change `Host` to `Address`？ Since the value always contains port information in the following.

Reply:
done